templates/layouts/aw-install-multi.hbs

```diff
@@ -31,6 +31,19 @@
         {{/with}}
         </table>
       {{/deviceNames}}
+      </table>
```

Comment:
I would suggest to put at least one spacing-row between the two tables. The hardware feature headings have various bottom spacings to the info table. Due to tilt-to-wake wrapping into 3 lines.
With spacing between the tables, the new model headings would be more obvious separated from the colored feature table.

Reply:
Good idea.  It does look better.
![image](https://user-images.githubusercontent.com/4911249/219118023-78fc213a-90f4-4489-b12c-2f550ddb340a.png)